Content.Shared/Storage/EntitySystems/DumpableSystem.cs

```diff
@@ -160,10 +160,11 @@ private void OnDoAfter(EntityUid uid, DumpableComponent component, DumpableDoAft
             dumped = true;
 
             var targetPos = _transformSystem.GetWorldPosition(args.Args.Target.Value);
+            var targetRot = _transformSystem.GetWorldRotation(args.Args.Target.Value);
```

Comment:
As above there's no reason to use the separate calls here.

Reply:
oh, I didn't know there was a combined function. thanks